package.json

```diff
@@ -30,12 +33,14 @@
     "eslint-release": "^3.2.0",
     "mocha": "^9.0.1",
     "opener": "^1.5.2",
-    "rollup": "^2.52.1"
+    "rollup": "^2.52.1",
+    "typescript": "^4.6.0-dev.20220206"
```

Comment:
Is it necessary for this feature to use this `-dev` version instead of a stable one?

Reply:
Apologies, left as a relic (along with `nodenext` resolution), both of which now reverted, which I believe I had been exploring due to problems with `outFile` and not finding the module issues.

(Using `outDir` turns out to have been the sole key for resolving that issue--with an apparently similar issue being reported, FWIW, by other users on [StackOverflow](https://stackoverflow.com/a/58534016/271577). That issue reports problems with module discovery when the output of `outFile` produces `declare module "index"`. Such output in this context is apparently what is responsible for preventing discovery of types by external projects (e.g., when I link my `espree` branch I'm working on to `eslint-visitor-keys`). I was only able to resolve this by use of `outDir`, which has the side effect of producing two declaration files instead of one).